### ISSUE TITLE
Hyper-V on 2013 Mac Pro is on but Lando installation says it's not

### ISSUE THREAD

**Issue by aangelinsf** [lando/lando#743]
**Revised:** I checked systeminfo and it appears that Hyper-V _is_ installed. Perhaps this is a Lando issue after all.

```
Hyper-V Requirements: 	VM Monitor Mode Extensions: Yes
                      	Virtualization Enabled In Firmware: Yes
                      	Second Level Address Translation: Yes
                      	Data Execution Prevention Available: Yes
```

Has anyone gotten the latest Lando (beta 35) to install on Win10 on Bootcamp? I have been unable to get it to work because the installer says Hyper-V is not enabled. I can't get it to work on Parallels 12, either, for what appears to be the same reason (Hyper-V not turned on).

I have:
1. Booted into macOS then into Windows. No luck. (Using the instructions in the link below.)
2. Followed these instructions to get the UEFI bootload installed:
https://apple.stackexchange.com/questions/120361/how-to-turn-on-hardware-virtualization-on-late-2013-macbook-pro-for-windows-8-1/234819#234819

Setup
------
2013 MacPro, 32GB, macOS 10.13.3
64GB Win10 Partition
Win10 Pro
lando-v3.0.0-beta.35.exe

**Comment by pirog** [comment]
@aangelinsf FWIW my Windows "test machine" is Win 10 Pro running on Bootcamp on an old mac mini. I have not seen something like this.

Not working on parallels doesnt surprise me since the VM inside VM use case can be tricky and is definitely not something we support.

**Comment by aangelinsf** [comment]
Mike, I did eventually get it running. This is my first time running Windows on Mac hardware via Bootcamp or attempting to use Hyper V inside Parallels. Usually I just run Parallels. It turns out there is another switch inside Windows itself that must be turned on to enable Hyper-V. 

I did get Lando/Docker to install and am moving to the next step. Fingers crossed (and I realize the Parallels environment isn't supported, will be trying both).
